pages/1.11/administering-clusters/convert-agent-type/index.md

```diff
@@ -8,18 +8,13 @@ excerpt: Learn to convert agent nodes to public or private agent nodes.
 enterprise: false
 ---
 
-<!-- The source repo for this topic is https://github.com/dcos/dcos-docs -->
-
-
-You can convert agent nodes to public or private for an existing DC/OS cluster.
-
-Agent nodes are designated as [public](/1.11/overview/concepts/#public-agent-node) or [private](/1.11/overview/concepts/#private-agent-node) during installation. By default, they are designated as private during [GUI][1] or [CLI][2] installation.
+You can convert agent nodes to public or private for an existing DC/OS cluster. Agent nodes are designated as [public](/1.11/overview/concepts/#public-agent-node) or [private](/1.11/overview/concepts/#private-agent-node) during installation. By default, they are designated as private during [installation](1.11/installing/evaluation/cloud-installation/).
 
 ### Prerequisites:
 These steps must be performed on a machine that is configured as a DC/OS node. Any tasks that are running on the node will be terminated during this conversion process.
 
-*   DC/OS is installed using the [custom](/1.11/installing/oss/custom/) installation method and you have deployed at least one [master](/1.11/overview/concepts/#master) and one [private](/1.11/overview/concepts/#private-agent-node) agent node.
-*   The archived DC/OS installer file (`dcos-install.tar`) from your [installation](/1.11/installing/oss/custom/gui/#backup).     
```

Comment:
/1.11/installing/#backup is incorrect. GUI and CLI installer pages are removed from 1.11 installing docs. Replace /1.11/installing/#backup with /1.11/installing/evaluation/cloud-installation/#backup